.env

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_API_BASE_URL=https://staging-api.learnhub.mk
+API_KEY=
```

Comment:
can you double check if we have the .env files in .gitignore
If not please add it.
documentation for reference:
https://git-scm.com/docs/gitignore

Reply:
I checked .gitignore and the .env file was already added. Is this enough, or should I add another one?

![IMG_20240429_173954.jpg](https://github.com/learnhubmkd/app/assets/97918443/f5006da9-73c5-4a5b-902c-e138c5cbcae2)

Reply:
So this would ignore files like: .env.production.local
But if we want to have .env ignored we need to add it. 
If it was ignored properly by .gitignore we be able to see it here, and comment on it 🗡️